### PR TITLE
DOC-4903: Pages and folders for collections, in temp location

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/introduction/pages/developer-preview/collections/cbstats-reference/cbstats-collections-details.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+----
+manifest:collection:0x0:name:                    _default
+manifest:collection:0x8:name:                    my_collection_in_my_scope_1
+manifest:collection:0x9:name:                    my_collection_in_my_scope_2
+manifest:collection:0xa:name:                    my_collection_in_default_scope
+manifest:collections:                            4
+manifest:default_exists:                         true
+manifest:uid:                                    4
+vb_0:collection:0x0:entry:end_seqno:             -6
```
Comment (jimwwalker) [inline modules/introduction/pages/developer-preview/collections/cbstats-reference/cbstats-collections-details.adoc]: Tony, I mentioned this output would be changing, the following is an example of the current output that will ship in the Developer Preview


`manifest:collection:0x0:name:                       _default`
`manifest:collection:0x8:name:                       my_collection_in_my_scope_1`
`manifest:collection:0x9:name:                       my_collection_in_my_scope_2`
`manifest:collection:0xa:name:                       my_collection_in_default_scope`
`manifest:collections:                               4`
`manifest:default_exists:                            true`
`manifest:uid:                                       4`
`vb_0:collection:0x0:entry:high_seqno:              26`
`vb_0:collection:0x0:entry:items:                   120`
`vb_0:collection:0x0:entry:persisted_high_seqno:    26`
`vb_0:collection:0x0:entry:scope:                   10`
`vb_0:collection:0x0:entry:start_seqno:             26`
.
.
.

Comment (jimwwalker) [reply modules/introduction/pages/developer-preview/collections/cbstats-reference/cbstats-collections-details.adoc]: basically end_seqno is removed and high_seqno is added

Comment (tonyjhillman) [reply modules/introduction/pages/developer-preview/collections/cbstats-reference/cbstats-collections-details.adoc]: I've modified the examples based on the latest output, from build 3506.

**Comment by amarantha-k** [comment]
The updates in this PR have been merged as part of https://github.com/couchbase/docs-server/pull/639.

**Pull request closed without merging**